libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html

```diff
@@ -30,10 +30,23 @@
     </small>
   </p>
 
-  <watt-description-list
-    *ngIf="processStepResults"
-    [groups]="getMetadata()"
-  ></watt-description-list>
+  <watt-description-list *ngIf="processStepResults">
+    <watt-description-list-item
+      [label]="t('meteringPointType')"
+      [value]="t('timeSeriesType.' + processStepResults.timeSeriesType)"
+    ></watt-description-list-item>
+    <watt-description-list-item
+      [label]="t('calculationPeriod')"
+      [value]="batch?.period | dhDateRange"
+    >
+    </watt-description-list-item>
+    <watt-description-list-item [label]="t('sum')" [value]="processStepResults.sum + ' kWh'">
```

Comment:
Question: In the old implementation, we can force a new row. Do you know how we can do this here? Should this be done by having two lists or a property on the `watt-description-list-item` component?

Reply:
You can set forceNewRow=true on a list item

Reply:
Perfect!